app/views/settings/_redmine_issues_tree.html.haml

```diff
@@ -1,5 +1,10 @@
 %p
   %label
-    = l(:'issues_tree.settings.default_redirect_to_tree_view')
-  = hidden_field_tag 'settings[default_redirect_to_tree_view]', false
-  = check_box_tag 'settings[default_redirect_to_tree_view]', true, @settings['default_redirect_to_tree_view'] == 'true'
+    = l(:'issues_tree.settings.default_redirect_to_issues_tree_view')
+  = hidden_field_tag 'settings[default_redirect_to_issues_tree_view]', false
```

Comment:
You can't change the name of the existed setting. In this case you drop the setting to default for all people who already use the plugin. You can create a migration for rename settings and save value of it, but this is not preferable way due to an unobvious working of a redmine settings.